src/benchmark/micro_benchmark_main.cpp

```diff
@@ -30,7 +30,7 @@
  * before the end of main().
  *
  *
- * ---> But similar things could happen anytime you call, e.g. StorageManager::get() before calling
+ * ---> But similar things could happen anytime you call, e.g. Hyrise::get().storage_manager before calling
```

Comment:
Please have a look at this whole comment and if it is still relevant. The static init order fiasco should be resolved by this PR. Maybe we should call `get_default_memory_resource()` before initializing any other class that was previously a singleton.